upgrade/upgrade-provider.go

```diff
@@ -176,7 +176,7 @@ func UpgradeProvider(ctx Context, repoOrg, repoName string) error {
 
 	var targetSHA string
 	if ctx.UpgradeProviderVersion {
-		repo.workingBranch = fmt.Sprintf("upgrade-terraform-provider-%s-to-v%s",
+		repo.workingBranch = fmt.Sprintf("upgrade-%s-to-v%s",
```

Comment:
`ctx.UpstreamProviderName` looks like `terraform-provider-%s`?

Reply:
Yes - we set it in the config file